taker/src/main.rs

```diff
@@ -383,9 +384,9 @@ async fn main() -> Result<()> {
                 routes::post_withdraw_request,
             ],
         )
-        .register("/api", rocket::catchers![rocket_basicauth::unauthorized])
+        .register("/api", default_catchers())
         .mount("/", rocket::routes![routes::dist, routes::index])
-        .register("/", rocket::catchers![rocket_basicauth::unauthorized])
+        .register("/", default_catchers())
```

Comment:
I think this can stay as is, we don't need the 422 catcher to serve the static assets.

Reply:
I agree with your reasoning that it might not be strictly necessary.
However, it's not wrong either - if 422 will not happen, then nothing gets caught.
It's easier to have them aligned than to always re-think whether it was needed or not.